gossip/src/crds.rs

```diff
@@ -871,7 +880,10 @@ mod tests {
     fn test_find_old_records_default() {
         let thread_pool = ThreadPoolBuilder::new().build().unwrap();
         let mut crds = Crds::default();
-        let val = CrdsValue::new_unsigned(CrdsData::LegacyContactInfo(ContactInfo::default()));
+        let val = {
+            let node = ContactInfo::new_localhost(&Pubkey::default(), /*now:*/ 1);
```

Comment:
Is my understanding correct that we previously would have failed this test when forcing `now` to 1, but after the old label filtering changes, we will still pass because we will use the lesser of peer timestamp and insertion timestamp (0 in this case)?

Reply:
The insertion timestamp is set in `crds.insert` in L888 which is `1`.
Previously the code was using `1` as the timestamp for the value.
But the new code will use `0` unless I add `/*now:*/ 1` when I create the value.
So without that the test fails in L893 (i.e. it deems the value as too old).